src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -158,6 +159,24 @@ public function load(array $configs, ContainerBuilder $container): void
             }
         }
 
+        if (!isset($config['cache']) || true === $config['cache']) {
+            $config['cache'] = new Reference('twig.template_cache.chain');
+
+            $cacheOptions = ($config['auto_reload'] ?? true) ? FilesystemCache::FORCE_BYTECODE_INVALIDATION : 0;
+            $container->getDefinition('twig.template_cache.runtime_cache')->replaceArgument(1, $cacheOptions);
```

Comment:
When `auto_reload` is true, the readonly build cache can be disabled, and `$config['cache'] = '%kernel.cache_dir%/twig'`.